packages/terra-signature/docs/DEPENDENCIES.md

```diff
@@ -0,0 +1,23 @@
+# Dependency Information
+
+## dependencies
```

Comment:
run npm install so this dependency doc gets regenerated with the updated dependencies.

Reply:
fixed: https://github.com/cerner/terra-core/pull/769/commits/3a8c2f78444b155b7b6ec315126022c03c9bb5d9